### PR TITLE
added section on release to conda-forge to developer guide

### PR DESCRIPTION
**Pull request by TonioF**
Closes https://github.com/dcs4cop/xcube/issues/382

### CODE REVIEW

File: docs/source/devguide.md
Comment (forman) [inline docs/source/devguide.md:295]:
```suggestion
* In `xcube/version.py` remove the `.dev` suffix from version name.
* Adjust version in `Dockerfile`  accordingly.
```

**Review by forman — CHANGES_REQUESTED** [review]
Thanks! But pls include `Dockerfile` version adjustment.

File: docs/source/devguide.md
Comment (TonioF) [reply docs/source/devguide.md:305]: Ok!

File: docs/source/devguide.md
Comment (pont-us) [inline docs/source/devguide.md:368]: I suggest an additional step here: merge the newly-merged changes from the master branch on conda-forge back to the master branch of the dcs4cop fork. This has to be done anyway at some point before the branch for the following release is created, but doing it immediately makes the current configuration more visible on the dcs4cop repo, and avoids messy parallel branches if someone subsequently checks changes directly into dcs4cop's master.

Comment (TonioF) [reply docs/source/devguide.md:368]: We could do so ... I don't think it's absolutely necessary and it would not replace the step that the author has to check that the branches on conda-forge and dcs4cop are identical, but your arguments are valid.

**Review by forman — APPROVED** [review]

File: docs/source/devguide.md
Comment (dzelge) [inline docs/source/devguide.md:326]: The pull request is automatically done by the regro-cf-autotick-bot  on release. The steps of creating a PR is only necessary if you need to update the current version i.e. build >0.

File: docs/source/devguide.md
Comment (dzelge) [inline docs/source/devguide.md:351]: Also this will be done by the bot automatically if build eq 0.

**Review by dzelge — CHANGES_REQUESTED** [review]
See comments

File: docs/source/devguide.md
Comment (TonioF) [reply docs/source/devguide.md:351]: Not clear. Do you argue that this sentence "It's also possible to have the rendering done by a bot as part of the pull request,
   but this doesn't seem to work very reliably in practice." is wrong?

Comment (pont-us) [reply docs/source/devguide.md:350]: I wrote that sentence based on my personal experience (see comments on https://github.com/conda-forge/xcube-feedstock/pull/9 ). It was some kind of permissions problem: the bot didn't have write access to commit its changes, and requested that I tick the "Allow edits from maintainers" box, but this box didn't even appear for me. I'm sure there was some way around it but I found it easier just to do the rerendering locally.

File: docs/source/devguide.md
Comment (pont-us) [reply docs/source/devguide.md:326]: I don't think that this applies in our case. [The conda-forge docs say](https://conda-forge.org/docs/maintainer/updating_pkgs.html#pushing-to-regro-cf-autotick-bot-branch):

> When a new version of a package is released on PyPI/CRAN/.., we have a bot that automatically creates version updates for the feedstock.

But we don't release on PyPI and the bot doesn't seem to track releases that only happen on GitHub. At least, I didn't notice any automatically created branches for any for the recent xcube / xcube-* releases.

(There's also the problem that we need to manually update dependencies for the conda releases, but I guess that would work if a bot created the PR too -- we just add our own changes before merging the PR.)

Comment (pont-us) [reply docs/source/devguide.md:326]: Oops, I was wrong: https://github.com/conda-forge/xcube-feedstock/pull/13 is indeed there for xcube 0.6.1. So evidently it does track GitHub releases after all.

**Pull request merged**
